src/modules/orders.ts

```diff
@@ -44,6 +45,7 @@ export namespace orders {
         if (!order) {
             order = new Order(orderId)
             order.isValid = true
+            order.isSigned = false
```

Comment:
I'm confused with this. Maybe I'm misremembering, but this should happen for smart contract orders that come via the PreSign event, right?

In that case, shouldn't the flag be set to `true`, and false otherwise?

Reply:
I belive its set to the correct value in `setPresignature` method. This is for `getOrCreateOrder` which correctly defaults to `false`. Then is overriden this default